aws/aws-cdk
# chore: re-enable using current pkg versions for api compat checks

This was originally added in commit
https://github.com/aws/aws-cdk/commit/3dd21b9b212f96720519ac12f9cb538c697e9343.

However, the script fails during a bump build when the package version
in lerna.json is ahead of the latest published in NPM. This was worked
around by turning this feature off -
https://github.com/aws/aws-cdk/commit/09a1f33f975e49f92bfa708e3d3d03984863a28d.

Re-enable this feature and handle version in lerna.json may be ahead of
NPM.

### Testing

Manually tested three cases -
* When version in `lerna.json` is <= package published in NPM
* When version in `lerna.json` is > package published in NPM
* When `DOWNLOAD_LATEST` is set to `true`.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*
